### ISSUE TITLE
Automated UI tests for sharing dialog fail if browser window is too narrow

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#28715]
### Steps to reproduce
1. Have a test system where the chrome browser opens quite narrow by default
2. Run the test for the shareeAutocompletion feature
```
bash tests/travis/start_behat_tests.sh --feature tests/ui/features/other/shareeAutocompletion.feature
```

### Expected behaviour
The test passes

### Actual behaviour
The test fails with:
```
    And the folder "simple-folder" is shared with the user "user1"                                                                        # SharingContext::theFileFolderIsSharedWithTheUser()
    And the share dialog for the folder "simple-folder" is open                                                                           # SharingContext::theShareDialogForTheFileFolderIsOpen()
      WebDriver\Exception\UnknownError: unknown error: Element <a class="action action-share permanent shared-style" href="#" data-action="Share" data-original-title="" title="">...</a> is not clickable at point (608, 318). Other element would receive the click: <div id="shareTabView" class="tab shareTabView">...</div>
        (Session info: chrome=60.0.3112.90)
        (Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.10.0-32-generic x86_64)
```

### Server configuration
**Operating system**:
Ubuntu 16.04
**Web server:**
PHP dev server
**Database:**
mySQL
**PHP version:**
7.0
**ownCloud version:** (see ownCloud admin page)
10.0.3 from master 2017-08-16

### Client configuration
**Browser:**
chrome 60.0.3112.90

**Operating system:**
Ubuntu 16.04

**Comment by phil-davis** [comment]
The problem is that the step:
```
And the folder "simple-folder" is shared with the user "user1"
```
opens the sharing dialog, does the requested sharing but then leaves the sharing dialog panel open.

The next step tries to click the icon to open the sharing dialog panel to do its thing, but the sharing dialog is already open and is obscuring the icon which would open it.

If the browser window is wider, then the "open sharing dialog panel" icon is visible and can be clicked (which works OK when the sharing dialog panel is already open).

So the test has only been working when the browser window is wide enough.

I have a fix coming...

**Comment by phil-davis** [comment]
Fixed in master #28716 and stable10 #28720
